HPXMLtoOpenStudio/measure.rb

```diff
@@ -1629,7 +1629,8 @@ def add_cooling_system(model, runner, weather, spaces, airloop_map)
 
         airloop_map[sys_id] = HVAC.apply_air_source_hvac_systems(model, runner, cooling_system, heating_system, sequential_cool_load_fracs, sequential_heat_load_fracs,
                                                                  weather.data.AnnualMaxDrybulb, weather.data.AnnualMinDrybulb,
-                                                                 conditioned_zone, @hvac_unavailable_periods, @schedules_file, @hpxml_bldg)
+                                                                 conditioned_zone, @hvac_unavailable_periods, @schedules_file, @hpxml_bldg,
+                                                                 spaces[HPXML::LocationConditionedSpace], @hpxml_header)
```

Comment:
We are already passing in the conditioned zone, you can get the space from that: `conditioned_zone.spaces[0]`